src/dodal/devices/hutch_shutter.py

```diff
@@ -0,0 +1,94 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import (
+    DEFAULT_TIMEOUT,
+    AsyncStatus,
+    StandardReadable,
+    wait_for_value,
+)
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+from dodal.log import LOGGER
+
+HUTCH_SAFE_FOR_OPERATIONS = 0  # Hutch is locked and can't be entered
+
+
+class ShutterNotSafeToOperateError(Exception):
+    def __init__(self, errmsg, *args: object) -> None:
+        LOGGER.error(errmsg)
+        super().__init__(*args)
```

Comment:
Should: The logger will automatically log any exceptions raised so I think we can just have this as:
```suggestion
class ShutterNotSafeToOperateError(Exception):
    pass
```